crates/iroha_cli/src/main.rs

```diff
@@ -1197,6 +1200,232 @@ mod json {
         }
     }
 }
+
+mod multisig {
+    use std::io::{BufReader, Read as _};
+
+    use executor_custom_data_model::multisig::{MultisigAccountArgs, MultisigTransactionArgs};
```

Comment:
I don't immediately like the idea of having this being included directly into the client. Though, it depends on how much we need to make `multisig` an official out-of-the-box feature.

As an alternative idea, there could a separate CLI to construct multisig-related instructions in a form of JSON, which _then_ could be supplied to the client:

```shell
iroha_multisig register --account .. --weights .. --quorum .. | iroha json transaction
```

Reply:
This can be discussed together with the client interface issues left to #4930